src/app/collection/constants.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+export const COLLECTION_NO_ICON = 'fa fa-exclamation-circle';
+
+export const COLLECTION_NO_COLOR = '#cccccc';
+
+export const COLLECTION_NO_CODE = 'COLLECTION_HAS_NO_CODE';
```

Comment:
I just wonder why we need this constant. Would not it be better if the default collection code was `null` or empty string and the server generated it? (It should be capable of it now.)

Reply:
Yes, that is the default behaviour, the `COLLECTION_NO_CODE` constant is there only to represent the default uninitialized collection in a map (and I wanted the map to be of type `string -> string` instead of `string | null -> string`).

Reply:
Then change its value to an empty string. It would better represent an uninitialized collection.